debian/compat

```diff
@@ -0,0 +1 @@
+10
```

Comment:
The compat level was set to 9 when generated by stdeb, I took the liberty to bump it to see if it works to silence deprecation warnings.